clients/python/README.md

```diff
@@ -36,3 +36,7 @@ When modifying templates on the [openapi-generator](https://github.com/ITISFound
 ```
 
 Hence, it is important to be able quickly modify the [templates](https://github.com/ITISFoundation/openapi-generator/tree/master/modules/openapi-generator/src/main/resources) and regenerate the client. To do this without rebuilding the docker image containing the openapi-generator, run `make python-client-from-templates path/to/template/dir` from `clients/python`.
+
+## Upcoming Deprecations
+
+In the future the `models` and `api` submodules of the `osparc` package will be deprecated. All functionality contained in there should instead by imported directly from `osparc`.
```

Comment:
My understanding is that something gets deprecated when you set the warning and retiring it is the next step. See this lifecycle:
https://user-images.githubusercontent.com/32402063/199943187-ae8136b9-91df-454b-86c4-21770b4afbe3.png

Therefore: `osparc.models` and `osparc.api` modules are deprecated and will be retired/removed in the next version, please used `osparc` to import all the functions instead. That is how i would phrase it

Reply:
Sure, I will change